src/components/molecules/DashboardItem.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Card, Pane, Heading, Paragraph } from 'evergreen-ui';
+
+export type DashboardItemProps = {
+    title: string;
+    titleSize: 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900;
+    width: number | string;
+    height: number | string;
+    paragraph?: string;
+    textSize?: 300 | 400 | 500;
+};
+
+const DashboardItem: React.FC<DashboardItemProps> = ({
+    title,
+    width,
+    paragraph,
+    height,
+    titleSize,
+    textSize,
+    children,
+}) => {
+    return (
+        <Card width={width} height={height} display="flex" flexDirection="column" border="default" elevation={1}>
+            <Heading size={titleSize} marginTop={25} marginLeft={25}>
+                {title}
+            </Heading>
+            {paragraph && textSize ? (
```

Comment:
Seems a bit strange to include the presence of the `textSize` property in the conditional. Could it perhaps have a default value of `300 | 400 | 500` and thus only include the `paragraph` in the conditional?

Reply:
True! I can add a default value instead